src/i18n.tsx

```diff
@@ -0,0 +1,15 @@
+import i18n from 'i18next';
```

Comment:
.tsx is only for files, which contain JSX syntax + Typescript.
If you use only typescript, then you should you .ts extension.

Reply:
Done.Thanks.